shelf_router/test/router_test.dart

```diff
@@ -148,6 +148,30 @@ void main() {
     expect(await get('/api/hello?ok'), 'middleware');
   });
 
+  test('mount(Router) does not require a trailing slash', () async {
+    var api = Router();
+    api.get('/', (Request request) {
+      return Response.ok('Hello World!');
+    });
```

Comment:
Can we make another end-point on `api`, so that we have test that this works for something like `/api/user/jonasfj/info`?